### PR TITLE
Update microsoft-365-monitoring.md

### PR DESCRIPTION
**Pull request by ematayur**
added one new description sentence towards the top of the page

### CODE REVIEW

File: microsoft-365/enterprise/microsoft-365-monitoring.md [hunk 1]
```diff
@@ -22,7 +22,11 @@ description: "Use Microsoft 365 monitoring for information about incidents or ad
 
 # Learn about Microsoft 365 monitoring
 
-You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability was initially started with Exchange Online and now getting expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps and more service in future. Monitoring provides you with information about incidents and advisories that are collected in these categories:
+You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability was initially started with Exchange Online and now getting expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps and more service in future.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-monitoring.md]:
```suggestion
You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability initially started with Exchange Online and is now being expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps, and more services in the future. 
```

File: microsoft-365/enterprise/microsoft-365-monitoring.md [hunk 2]
```diff
@@ -22,7 +22,11 @@ description: "Use Microsoft 365 monitoring for information about incidents or ad
 
 # Learn about Microsoft 365 monitoring
 
-You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability was initially started with Exchange Online and now getting expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps and more service in future. Monitoring provides you with information about incidents and advisories that are collected in these categories:
+You can use dashboards in the [Microsoft 365 admin center](https://go.microsoft.com/fwlink/p/?linkid=2024339) to monitor the health of various Microsoft services for your organization's Microsoft 365 subscription. This capability was initially started with Exchange Online and now getting expanded to other Microsoft services like Microsoft Teams, Microsoft 365 Apps and more service in future. 
+
+Microsoft 365 Monitoring enables increases observability and minimizes downtime through providing near real time, user usage telemetry data with enriched alerts in M365 Admin Center's Service Health dashboard.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-monitoring.md]:
```suggestion
Microsoft 365 monitoring increases observability and minimizes downtime by providing near real-time, user usage telemetry data with enriched alerts in Microsoft 365 admin center Service Health dashboard. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ematayur, kindly commit the suggested changes. Many thanks.

File: microsoft-365/enterprise/microsoft-365-monitoring.md (hunk 1 above)
Comment (temanteleary) [reply microsoft-365/enterprise/microsoft-365-monitoring.md]: @JohanFreelancer9 Please change the first part of the last sentence to "This capability initially started with Exchange Online and now is being expanded to other Microsoft services . . . "

Comment (JohanFreelancer9) [reply microsoft-365/enterprise/microsoft-365-monitoring.md]: Done. @ematayur, please commit these suggestions. Many thanks.

Comment (ematayur) [reply microsoft-365/enterprise/microsoft-365-monitoring.md]: thank you for the feedback. committed with the suggestions

**Pull request merged**